02.calendar/calender.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+require "optparse"
+require "date"
+
+# ---------------------------------------------------
+# コマンドラインの引数で指定された月（-m）と年（-y）を取得する
+# ---------------------------------------------------
+params = ARGV.getopts("m:", "y:")
+
+# 指定された月（-m）をmonthに格納する（指定がない場合は今月）
+if params["m"] == nil
+  month = Date.today.month
+else
+  month = params["m"].to_i
+end
```

Comment:
あえて`== nil`は書かずに、あと三項演算子を使って、こんなふうに書くこともできます。

```suggestion
month = params["m"] ? params["m"].to_i : Date.today.month
```

Reply:
書いてみて、めちゃくちゃスッキリしました！
三項演算子、自分が読み解くのはまだあまり自信がないですが、プロっぽくてかっこいいので積極的に使っていこうと思いました！